### ISSUE TITLE
Replace Spongycastle with bouncycastle or use android built in version

### ISSUE THREAD

**Issue by TuxCoder** [austrianredcross/stopp-corona-android#22]
Spongycastel is a fork from bouncycastle.
This fork was created because of:
```
The Android platform unfortunately ships with a cut-down version of Bouncy Castle - as well as being crippled, it also makes installing an updated version of the libraries difficult due to classloader conflicts.
```
Src: https://rtyley.github.io/spongycastle/

The thing is this, Android resolved this confliction after Android 3.
As this libary is not needed anymore and it needs time to maintain.
The developer canceled the support!

Ref: https://github.com/rtyley/spongycastle/issues/34
Ref2: https://github.com/open-keychain/open-keychain/issues/1676

As result the latest updated was 2017-08-30.

Please replace this package with a upstream bouncycastle or use the android build in `com.android.org.bouncycastle` if possible.

**Comment by mitsuhiko** [comment]
I think this one might be worth looking into @uluckas.

**Comment by uluckas** [comment]
Fixed in bc35c4384f05d4dfd5562aed98c0b0aaa905f782
